apps/dotcom/styles/core.css

```diff
@@ -1,8 +1,17 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@500;600;800&display=swap');
 
 :root {
-	font-family: Inter, -apple-system, 'system-ui', 'Segoe UI', 'Noto Sans', Helvetica, Arial,
-		sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji';
+	font-family:
+		Inter,
+		-apple-system,
```

Comment:
drive-by nit: `system-ui` is prefered over `-apple-system` these days:
https://css-tricks.com/snippets/css/system-font-stack/

besides, this will help fix that serif problem we were talking about @steveruizok with the apple-system giving a serif font (yuck)